src/brpc/rdma/rdma_endpoint.cpp

```diff
@@ -513,7 +520,10 @@ void* RdmaEndpoint::ProcessHandshakeAtClient(void* arg) {
             std::min(ep->_sq_size, remote_msg.rq_size) - RESERVED_WR_NUM;
         ep->_remote_window_capacity = 
             std::min(ep->_rq_size, remote_msg.sq_size) - RESERVED_WR_NUM,
-        ep->_window_size.store(ep->_local_window_capacity, butil::memory_order_relaxed);
+        ep->_remote_rq_window_size.store(
```

Comment:
why _remote_rq_window_size not equal to _remote_window_capacity

Reply:
ibv_post_send is limited by the local sq and the remote rq, but _remote_window_capacity is the minimum value of the local rq and the remote sq, so the maximum value of _remote_rq_window_size should not be equal to _remote_window_capacity.